website/frontend/src/api/initApi.ts

```diff
@@ -12,3 +12,25 @@ export function initApi() {
   }
   return gear;
 }
+
+class NodeApi {
+  get api(): GearApi | null {
+    // eslint-disable-next-line no-underscore-dangle
+    return this._api;
+  }
+
+  private readonly address;
+
+  private _api: GearApi | null = null;
+
+  constructor(address = 'ws://localhost:9944') {
+    this.address = address;
+  }
+
+  async init() {
+    // eslint-disable-next-line no-underscore-dangle
+    this._api = await GearApi.create({ providerAddress: this.address });
+  }
+}
+
+export const nodeApi = new NodeApi();
```

Comment:
Add an env var here

Reply:
ok, could you tell me what kind of env variable?